gnovm/pkg/gnolang/uverse_test.go

```diff
@@ -158,3 +158,247 @@ func TestIssue1337PrintNilSliceAsUndefined(t *testing.T) {
 		})
 	}
 }
+
+func TestIssue2707PointerSliceAsParamInLen(t *testing.T) {
```

Comment:
These are fancy filetests; please move these to gnovm/tests/files. (If you want a nice chore, the above could also just be moved to filetests).

Reply:
https://github.com/gnolang/gno/pull/2709/commits/af7364186e2aea805b41350320a477f371da3a57